maintainers/maintainer-list.nix

```diff
@@ -17728,6 +17728,12 @@
     githubId = 35628088;
     name = "oluceps";
   };
+  oluchitheanalyst = {
+    name = "ndukauba maryjane oluchi";
+    email = "ndukauba.oluchi2000@gmail.com";
+    github = "oluchitheanalyst";
+    githubId = 152318870;
+  };
```

Comment:
Adding yourself to the maintainers list needs to be in its own commit (see the [commit conventions](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions)):

> When adding yourself as maintainer in the same pull request, make a separate commit with the message maintainers: add <handle>. Add the commit before those making changes to the package or module. See [Nixpkgs Maintainers](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) for details.